.changeset/tender-adults-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@statelyai/inspect": patch
+---
+
+You can now use `@statelyai/inspect` without installing `xstate`.
```

Comment:
This might not be 100% true with this patch. The runtime might work OK but the TypeScript would complain at you because some of those types "leak" to the consumer of `@statelyai/inspect`

Reply:
besides that, `xstate` is still specified as a peer dependency here. This is a signal for the consumer that `xstate` is actually required to run this package